### ISSUE TITLE
Sample app not searching

### ISSUE THREAD

**Issue by melanieberrut** [besarthoxhaj/bes-tube#1]
I followed the following steps as per guide:
```
  $ yarn install
  $ yarn dev
```
The app loads on `http://localhost:3000/`, but searching doesn't work - here is the error:
`index.js:38 POST http://localhost:9000/api/search net::ERR_CONNECTION_REFUSED`

If I run `yarn start` to get the server running on port `9000`, I get `index.js:38 POST http://localhost:9000/api/search 400 (Bad Request)`

**Comment by besarthoxhaj** [comment]
Readme updated.

Let me know if you mange to make it work!:)

**Comment by melanieberrut** [comment]
Nope, still an issue - I have my key in .env file as described. Still getting 400 bad request.

**Comment by besarthoxhaj** [comment]
Ah! That's not good

**Comment by besarthoxhaj** [comment]
This is probably where you get your error: https://github.com/besarthoxhaj/bes-tube/blob/master/api/routes.js#L90


Below a short snipped; 
```js
youtube.search.list({
  part: 'snippet',
  q: payload['text'],
}, (err, data) => {

  // this `if` statements gets tiggered
  // by the error on the YouTube req.
  // Try to put a `console.log(err)`
  // and tell me what you find
  if(err) {
    return res({
      status: 'error',
      data: err,
    }).code(400);
  }

  return res({
    status: 'success',
    data: data,
  });
});
```

Is likely due to something missing during the Google Dev registration.
- check if you enable the YouTube API to work with your app
- the `.env` looks correct
